### PR TITLE
Tickets/dm26112

### PR DESCRIPTION
**Pull request by rbovill**
Added the infrastructure necessary to run the Conda packaging job.

### CODE REVIEW

**Comment by tribeiro** [comment]
Do you mind squashing the commits into a single commit?

File: Jenkinsfile.conda [hunk 1]
```diff
@@ -0,0 +1,98 @@
+pipeline {
+    agent{ 
+        docker {
+            image 'lsstts/conda_package_builder:latest'
+            alwaysPull 'true'
+            args "--env TS_CONFIG_OCS_DIR=/home/saluser/ts_config_ocs --env LSST_DDS_DOMAIN=citest -u root --entrypoint=''"
+        }
+    }
+    environment {
+        package_name = "ts_environment"
```
Comment (tribeiro) [inline Jenkinsfile.conda]: Package_name is set to `ts_environment` should probably be `environment` only as the `ts-` is added already in later stages (check line 81).

**Review by tribeiro — CHANGES_REQUESTED** [review]
Spotted a small bug in the Jenkinsfile. It probably passed the [build](https://tssw-ci.lsst.org/view/Conda%20Jobs/job/Environment%20conda%20package/job/tickets%252FDM-26112/6/console) on the branch because it skipped the upload step.

> Stage "Push Conda Release package" skipped due to when conditional

File: Jenkinsfile.conda (hunk 1 above)
Comment (rbovill) [reply Jenkinsfile.conda]: Fixed.

**Review by tribeiro — APPROVED** [review]

File: Jenkinsfile.conda [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            when {
+                buildingTag()
+            }
+            steps {
+                withCredentials([usernamePassword(credentialsId: 'CondaForge', passwordVariable: 'anaconda_pass', usernameVariable: 'anaconda_user')]) {
+                    withEnv(["HOME=${env.WORKSPACE}"]) {
+                        sh """
+                        source /home/saluser/miniconda3/bin/activate
+                        anaconda login --user ${anaconda_user} --password ${anaconda_pass}
+                        anaconda upload -u lsstts --force ~/miniconda3/conda-bld/linux-64/ts-${package_name}*.tar.bz2
```
Comment (tribeiro) [inline Jenkinsfile.conda]: I think this needs to be 

```
anaconda upload -u lsstts --force /home/saluser/miniconda3/conda-bld/linux-64/ts-${package_name}*.tar.bz2
```

Comment (rbovill) [reply Jenkinsfile.conda]: Using the full path name, instead of the shortcut.

**Pull request merged**